contributing/review-process.md

```diff
@@ -38,6 +38,40 @@ In The Falco Project, we strive to foster a positive and constructive environmen
 - When expressing your code style preferences, please prioritize specific style modifications that bring clear benefits to the project in terms of performance or maintainability. It is important to avoid excessive emphasis on personal style. This approach promotes inclusivity and respect for different coding styles, as long as they align with the project's best interests.
 - Please remember that in the realm of security, striking a balance between speed, caution, and code reorganization is crucial, considering that time is of the essence. This balanced approach allows us to effectively support the necessary capabilities for enhanced threat detection, performance, and scalability benefits within reasonable timelines.
 
+## How the Review Process Works
+
+ - The author submits a PR.
+
+- Automation ([@poiana](https://github.com/poiana)) suggests reviewers for the PR:
+
+  - Determine the set of [OWNERS](https://github.com/falcosecurity/evolution/blob/main/GOVERNANCE.md#repository-ownership) files.
+  - Choose at least two suggested *reviewers*, trying to find a unique reviewer for every leaf OWNERS file, and request their reviews on the PR. 
+- Humans review the PR:
+   - Reviewers look for general code quality, correctness, style and follow [reviewers guideline](#guidelines-for-code-reviewers)
+   - Anyone in the determined set of OWNERS (including both `reviewers` and `approvers`) can act as a reviewer, with the exception of the individual who opened the PR.
+   - If the code changes look good to them, a reviewer types `/lgtm` in a PR comment or selects the _Approved_ checkbox and clicks the _Submit review_ button on the GitHub UI; if they change their mind, they can use `/lgtm cancel.`
+   - Once a reviewer has `/lgtm`'ed, [@poiana](https://github.com/poiana) applies an `lgtm` label to the PR.
```

Comment:
I would just remove these bullet points wrt lgtm as we outline this in the following section.